docs/_pages/2.modularize-schema.md

```diff
@@ -49,12 +49,12 @@ From our index.js file, we can cut and paste the contents of the gql tag definin
 file. You’ll notice that we will no longer use the gql tag. Instead, we export a single constant that is our typeDef as just a string using template string
 syntax for easy multi-line support. Many IDEs have plugins and extensions available to aid in the development of graphql types. Installing and enabling can be
 useful for development. By adding the `/* GraphQL */` identifier before our template string, our IDE can perform proper syntax highlighting if an appropriate
-extension is loaded.
+extension is loaded. Using VS Code I prefer [Apollo GraphQL](https://marketplace.visualstudio.com/items?itemName=apollographql.vscode-apollo).
```

Comment:
would it make sense to create a new section in https://github.com/hydrateio/advanced-graphql-server-tutorial/blob/master/docs/_pages/100.resources.md and add this there, too?

Reply:
yeah, I created #40 for it.